### ISSUE TITLE
source code styling

### ISSUE THREAD

**Issue by KitWallace** [KitWallace/XQuery-Wikibook#7]
The book mostly encloses XQuery source code in source tags with lang set to xml, but the
wikibook platform now  supports xquery as a language with rather good syntax colouring .

see https://en.wikibooks.org/w/index.php?title=XQuery/HelloWorld

**Comment by lguariento** [comment]
That's cool. Is there any way we can get some sort of result such as the execution/visualisation here: http://exist-db.org/exist/apps/demo/examples/basic/hello.html ? It'd be great to have a similar style (and/or even have an eXist-db/eXide engine behind the scenes!)

**Comment by KitWallace** [comment]
Yes see Issue 9  which also points to another project of mine which compares different approaches to common tasks  -  which does not use exide but does have a common source for display and execution

http://kitwallace.co.uk/Book/ 

A web site using exide would be possible but I don't see how to combine this approach with a Wikibook and all its mechanisms for version tracking, public engagement, search, indexing etc.  Someone with better understanding of the wikibook platform may be able to  help with this integration.  There is obviously a security issue if the public were able to edit and then execute arbitrary code on someones server so this seems incompatible with the wikibook ethos - this is possible in mediawiki but I dont know if that's possible in wikibooks.

Failing that, we would to develop our own wikibook-like environment with restrictive editing. I'm all for a 'literate programming' environment but developing it takes us away from writing an XQuery wikibook
